code-studio/src/css/plc/preview_evaluation.scss

```diff
@@ -0,0 +1,12 @@
+.preview_evaluation {
+  display: flex;
```

Comment:
Have you checked this page in IE?  It looks like their `display: flex;` support may be spotty: http://caniuse.com/#feat=flexbox